handbook/communication/content_guidelines/style_and_mechanics.md

```diff
@@ -557,6 +557,25 @@ Use periods in complete sentences. Don’t use periods in headlines or labels. P
 
 ### Quotation marks
 
+When quoting content, use double quotation marks. Use single quotation marks to quote content within an existing quote.
```

Comment:
Might be worth giving this tip?
```suggestion
When quoting content, use double quotation marks. Use single quotation marks to quote content within an existing quote.<br>
In HTML, using `<q></q>` will automatically apply the right quotation marks for nested `<q>` tags.
```

Reply:
Ooh, good call. Just made one extra tweak for clarity.

```suggestion
When quoting content, use double quotation marks. Use single quotation marks to quote content within an existing quote.
In HTML, using `<q></q>` will automatically apply the correct quotation marks for nested `<q>` tags.
```